rct229/data_fns/table_G3_4_fns.py

```diff
@@ -0,0 +1,79 @@
+import rct229
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+
+# This dictionary maps the opaque surface type to construction enumerations to
+# the corresponding construction values in ashrae_90_1_prm_2019.construction_properties.json
+OPAQUE_SURFACE_TYPE_TO_CONSTRUCTION_MAP = {
+    "ABOVE-GRADE WALL": "PRM Steel Framed Exterior Wall",
+}
+
+# This dictionary maps the surface conditioning category enumerations to
+# the corresponding building category values in ashrae_90_1_prm_2019.construction_properties.json
+SURFACE_CONDITIONING_CATEGORY_TO_BUILDING_CATEGORY_MAP = {
+    "EXTERIOR RESIDENTIAL": "Residential",
+    "EXTERIOR NON-RESIDENTIAL": "Nonresidential",
+    "SEMI-EXTERIOR": "Semiheated",
+}
+
+# This dictionary maps the ClimateZone2019ASHRAE901 enumerations to
+# the corresponding climate zone set values  in the OSSTD file ashrae_90_1_prm_2019.construction_properties.json
+CLIMATE_ZONE_ENUMERATION_TO_CLIMATE_ZONE_MAP = {
```

Comment:
This should be `CLIMATE_ZONE_ENUMERATION_TO_CLIMATE_ZONE_SET_MAP` since the matching field in the OSSTD table is `climate_zone_set`.